client/client.go

```diff
@@ -655,6 +655,28 @@ func (client *Client) CreateUsers(options []*CreateUserOptions) ([]*CreateUserRe
 	return results, nil
 }
 
+// RemoveUserOptions holds options for removing a local system user.
+type RemoveUserOptions struct {
+	Username string `json:"username,omitempty"`
```

Comment:
Nitpick: do we expect the username to be empty? This is the sole field in the structure so that seems unusual.

Reply:
Indeed. Fixing that.